Documentation/Cluster.rst

```diff
@@ -21,7 +21,7 @@ To view the status of the cluster, run the following on an MCVirt node:
     
 
 
-This will show the cluster nodes, IP addresses, and status.
+This will show the cluster nodes, IP addresses, and status. If the IP address field for the local node is blank then you must set ``cluster_ip`` to the node's IP address in the configuration file.

```

Comment:
Maybe put the comment about the configuration change in the documentation for adding a new machine to the cluster?
Or a new section just before adding a node, like 'Initial node configuration' or something?